__tests__/nc_news_test.test.js

```diff
@@ -159,7 +176,15 @@ describe('GET /api/topics', () => {
             expect(response.body.msg).toBe("Path Not Found");
           })  
         })
-})
+        test('Responds with a 404 status code, when requesting a topic that doesn\'t exist', () => {
+            return request(app)
+            .get("/api/articles?topic=philosophy")
+            .expect(404)
+            .then((response)=>{
+                expect(response.body.msg).toBe("No articles found")
+            })
+        }); 
+    });
```

Comment:
I'd expect to see a 200 test for a valid topic that has no articles. I believe the paper topic has no articles linked to it.